official/eap/templates/eap-xp4-basic-s2i.json

```
@@ -0,0 +1,531 @@
+			"iconClass": "icon-eap",
+			"openshift.io/display-name": "JBoss EAP XP 4.0 basic template",
+			"openshift.io/provider-display-name": "Red Hat, Inc.",
+			"tags": "eap,eap-xp,javaee,java,jboss",
+			"template.openshift.io/documentation-url": "https://access.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/",
+			"template.openshift.io/long-description": "This template defines resources needed to develop a JBoss Enterprise Application Platform XP based application, including a build configuration, application deployment configuration and secure communication using edge TLS.",
+			"template.openshift.io/support-url": "https://access.redhat.com",
+			"version": "1.0"
+		}
+	},
+	"message": "A new JBoss EAP XP based application has been created in your project.",
+	"objects": [
+		{
+			"apiVersion": "v1",
+			"kind": "Service",
+			"metadata": {
+				"annotations": {
+					"description": "The web server's http port."
+				},
+				"labels": {
+					"application": "${APPLICATION_NAME}"
+				},
+				"name": "${APPLICATION_NAME}"
+			},
+			"spec": {
+				"ports": [
+					{
+						"port": 8080,
+						"targetPort": 8080
+					}
+				],
+				"selector": {
+					"deploymentConfig": "${APPLICATION_NAME}"
+				}
+			}
+		},
+		{
+			"apiVersion": "v1",
+			"kind": "Service",
+			"metadata": {
+				"annotations": {
+					"description": "The JGroups ping port for clustering.",
+					"service.alpha.kubernetes.io/tolerate-unready-endpoints": "true"
+				},
+				"labels": {
+					"application": "${APPLICATION_NAME}"
+				},
+				"name": "${APPLICATION_NAME}-ping"
+			},
+			"spec": {
+				"clusterIP": "None",
+				"ports": [
+					{
+						"name": "ping",
+						"port": 8888
+					}
+				],
+				"publishNotReadyAddresses": true,
+				"selector": {
+					"deploymentConfig": "${APPLICATION_NAME}"
+				}
+			}
+		},
+		{
+			"apiVersion": "v1",
+			"id": "${APPLICATION_NAME}-https",
+			"kind": "Route",
+			"metadata": {
+				"annotations": {
+					"description": "Route for application's https service."
+				},
+				"labels": {
+					"application": "${APPLICATION_NAME}"
+				},
+				"name": "${APPLICATION_NAME}"
+			},
+			"spec": {
+				"tls": {
+					"insecureEdgeTerminationPolicy": "Redirect",
+					"termination": "edge"
+				},
+				"to": {
+					"name": "${APPLICATION_NAME}"
+				}
+			}
+		},
+		{
+			"apiVersion": "image.openshift.io/v1",
+			"kind": "ImageStream",
+			"metadata": {
+				"labels": {
+					"application": "${APPLICATION_NAME}"
+				},
+				"name": "${APPLICATION_NAME}"
+			}
+		},
+		{
+			"apiVersion": "image.openshift.io/v1",
+			"kind": "ImageStream",
+			"metadata": {
+				"labels": {
+					"application": "${APPLICATION_NAME}"
+				},
+				"name": "${APPLICATION_NAME}-build-artifacts"
+			}
+		},
+		{
+			"apiVersion": "v1",
```

Please fix in your upstream:
```suggestion
			"apiVersion": "build.openshift.io/v1",
```